Memory/memory.cs

```diff
@@ -1414,43 +1414,47 @@ public void CloseProcess()
         /// <summary>
         /// Inject a DLL file.
         /// </summary>
-        /// <param name="strDllName">path and name of DLL file.</param>
+        /// <param name="strDllName">path and name of DLL file.</param>return;
         public void InjectDll(String strDllName)
         {
-            IntPtr bytesout;
+	        if (theProc == null 
+	            || pHandle == IntPtr.Zero
+	            || !File.Exists(strDllName))
+		        return;
 
-            foreach (ProcessModule pm in theProc.Modules)
-            {
-                if (pm.ModuleName.StartsWith("inject", StringComparison.InvariantCultureIgnoreCase))
-                    return;
-            }
+	        if (theProc.Modules.Cast<ProcessModule>().Any(pm => pm.ModuleName.StartsWith("inject", StringComparison.InvariantCultureIgnoreCase))) 
```

Comment:
you can just use OfType linq method

https://stackoverflow.com/questions/4015930/when-to-use-cast-and-oftype-in-linq